src/index.html

```diff
@@ -7,6 +7,341 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    
+    <header class="header">
+      <div class="header__catalog logo">
+        <img 
+          src="./images/logo.png" 
+          alt="moyo" 
+        >
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="Apple" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="Samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="Smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="Laptops & Computers" 
+              class="nav__link" 
+              data-qa="nav-hover"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="Gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="Tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="Photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="Video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__card" data-qa="card">
+        <div class="card card--container">
+          <img 
+            src="./images/imac.jpeg" 
+            class="card__image"
+            alt="imac"
+          >
+        
+          <h1 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+
+          <p class="card__code"> Product code: 195434 </p>
+
+          <div class="card__rating">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
```

Comment:
Set default modilfier. Fix all places
```suggestion
            <div class="stars stars--4">
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
            </div>
```